tests/addcomment.test.js

```diff
@@ -0,0 +1,22 @@
+import addComment from '../src/module/addcomment.js';
+
+global.fetch = jest.fn(() => Promise.resolve({
+  text: () => Promise.resolve('created'),
+}));
+
+describe('Test Add Comment Function', () => {
+  it('Add Comment ', async () => {
+    const status = await addComment(1);
+    expect(status).toBe('created');
+  });
+
+  it('Number of Fetch call 1', async () => {
+    await addComment(1);
+    expect(fetch).not.toBeCalledTimes(3);
+  });
+
+  it('Number of Fetch call', async () => {
+    await addComment(1);
+    expect(fetch).toBeCalledTimes(3);
+  });
+});
```

Comment:
- [ ] Please, consider that this test did not pass:
<img width="1018" alt="Screen Shot 2022-09-03 at 17 11 03" src="https://user-images.githubusercontent.com/49064106/188271944-c7a29063-122b-46b6-8f06-fd6a14bf02da.png">